### PR TITLE
GameSettings: Remove forced slow depth from fixed titles.

### PR DESCRIPTION
**Pull request by Armada651**
Each game has been tested and is confirmed working with fast depth. There are no more instances of forced slow depth in the Game INIs.

I also added a small cosmetic change in TextureConversionShader I came across to this PR.

---
This change is [Reviewable](https://reviewable.io/reviews/dolphin-emu/dolphin/4147)

### CODE REVIEW

**Comment by Armada651** [comment]
I looked at the pixel shader in the graphics debugger again, rounding is not accurate when compared with fast depth. I will remove that commit and further examine the hardware test in a graphics debugger.

**Comment by JosJuice** [comment]
Please remove the [Video_Settings] section from the INIs where that section now is empty.

File: Source/Core/VideoCommon/VideoCommon.h [hunk 1]
```diff
@@ -29,7 +29,7 @@ const u32 MAX_XFB_WIDTH = 720;
 // Although EFB height is 528, 574-line XFB's can be created either with
 // vertical scaling by the EFB copy operation or copying to multiple XFB's
 // that are next to each other in memory (TODO: handle that situation).
-const u32 MAX_XFB_HEIGHT = 574;
+const u32 MAX_XFB_HEIGHT = 576;
```
Comment (BhaaLseN) [inline Source/Core/VideoCommon/VideoCommon.h]: Is that change intentional? If yes, what about the comment above that also includes the number?

Comment (Armada651) [inline Source/Core/VideoCommon/VideoCommon.h]: Oops, not intentional, debug code I accidentally committed. Thanks for point that out.

**Comment by Armada651** [comment]
Reviewed 12 of 14 files at r1, 1 of 2 files at r2.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/dolphin-emu/dolphin/4147)*

**Comment by JMC47** [comment]
LGTM regarding the games I can test.
